<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StalledIssuesDialog</class>
 <widget class="QDialog" name="StalledIssuesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>815</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stalled Issues</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QStackedWidget
{
    border-top: 0px;
    border-bottom: 0px;
    border-left: 0px;
    border-right: 0px;
}

#EmptyViewContainerPage QToolButton
{
    border:none;
}

#TreeViewContainer{
background-color: #FFFFFFFF; /*colorToken.page-background*/
}

#TreeViewContainer QTreeView{
background-color: #FFFFFFFF; /*colorToken.page-background*/
border:none;
}

/**********  TreeView Container **********/
QScrollBar:vertical
{
    background: #f6f6f6;
    width: 15px;
    border-left: 1px solid #E4E4E4;
}

QScrollBar::sub-line:vertical, QScrollBar::add-line:vertical
{
    border: none;
    background: none;
    width:0px;
    height:0px;
}

QScrollBar::handle:vertical
{
    background: #c0c0c0;
    min-height: 20px;
    border-radius: 4px;
    margin: 3px 4px 3px 2px;
}

QMenu
{
    border: 1px solid #B8B8B8;
   border-radius: 8px;
    background: #ffffff;
    padding-top: 16px;
    padding-bottom: 16px;
    padding-left: 8px;
    padding-right: 8px;
}

QMenu::item
{
   font-size: 14px;
   color: #333333;
   background: transparent; 
   margin-top: 3px;
   margin-bottom: 3px;
   padding-left: 16px;
   padding-right: 16px;
   margin-left: 16px;
   margin-right: 16px;
}

QMenu::item:selected
{
    font-weight: bold;  
}

QMenu::separator
{
    height: 1px;
	width: 164px;
	min-width: 164px;
	max-width: 164px;
    background-color: #EEEEEE;
    margin-top: 9px;
    margin-bottom: 9px;
}

QMenu::item:disabled
{
    background-color: #ffffff;
    color: rgba(119,119,119,0.3);
}

QMenu::item:default
{
    color: #F46265;
}

/**********  TreeView Container **********/

/**********  Footer **********/
#footer[scroll=true]
{
    background-color: #FFAAAAAA; /*colorToken.surface-1*/
}

#footer[scroll=false]
{
    background-color: #FFFFFFFF; /*colorToken.page-background*/
}

#HelpButton {
border: none;
border-color: transparent;
color: #FF0078D7;/*colorToken.link-primary*/
}

#HelpButton QPushButton::!enabled {
color: grey;
}
/**********  Footer **********/

/**********  Header **********/

#header
{
    background-color: #FFAAAAAA; /*colorToken.surface-1*/
    border:  1px solid #5254570D; /*colorToken.border-subtle*/
}

#titleLabel
{
  font-size:18px;
  font-weight: 700;
  color: #FFF3F4F4; /*colorToken.text-primary*/
}

#title QPushButton {
    background-color: none;
    outline: none;
    border: none;
}
/**********  Header **********/

 </string>
  </property>
  <property name="TOKENIZED" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="header" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>88</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>88</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>23</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <item>
       <widget class="QWidget" name="title" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="titleLabel">
           <property name="text">
            <string>Sync issues</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="refreshButton">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../gui/Resources_light.qrc">
             <normaloff>:/refresh_issues</normaloff>:/refresh_issues</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SettingsButton">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../gui/Resources_light.qrc">
             <normaloff>:/settings</normaloff>:/settings</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="categories" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>26</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="StalledIssueTab" name="allIssuesTab">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="title" stdset="0">
            <string/>
           </property>
           <property name="filterCriterion" stdset="0">
            <number>0</number>
           </property>
           <property name="iconName" stdset="0">
            <string notr="true">tab_all_issues</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StalledIssueTab" name="nameConflictsTab">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="title" stdset="0">
            <string/>
           </property>
           <property name="filterCriterion" stdset="0">
            <number>1</number>
           </property>
           <property name="iconName" stdset="0">
            <string notr="true">tab_name_conflicts</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StalledIssueTab" name="itemTypeTab">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="title" stdset="0">
            <string/>
           </property>
           <property name="filterCriterion" stdset="0">
            <number>2</number>
           </property>
           <property name="iconName" stdset="0">
            <string notr="true">tab_item_type_conflicts</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StalledIssueTab" name="otherTab">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="title" stdset="0">
            <string/>
           </property>
           <property name="filterCriterion" stdset="0">
            <number>3</number>
           </property>
           <property name="iconName" stdset="0">
            <string notr="true">tab_others</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StalledIssueTab" name="failedTab">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="title" stdset="0">
            <string/>
           </property>
           <property name="filterCriterion" stdset="0">
            <UInt>4</UInt>
           </property>
           <property name="iconName" stdset="0">
            <string notr="true">tab_failed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StalledIssueTab" name="solvedIssuesTab">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="title" stdset="0">
            <string/>
           </property>
           <property name="filterCriterion" stdset="0">
            <UInt>5</UInt>
           </property>
           <property name="iconName" stdset="0">
            <string notr="true">tab_resolved</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="TreeViewContainer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="TreeViewContainerPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="StalledIssuesView" name="stalledIssuesTree">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="autoScroll">
          <bool>false</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="headerCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="EmptyViewContainerPage">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../gui/Resources_common.qrc">
           <normaloff>:/images/transfer_manager/empty_categories/no_transfers@2x.png</normaloff>:/images/transfer_manager/empty_categories/no_transfers@2x.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>240</width>
           <height>240</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="footer" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>18</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="HelpButton">
        <property name="text">
         <string>Help</string>
        </property>
        <property name="icon">
         <iconset resource="../../../gui/Resources_common.qrc">
          <normaloff>:/help-circle</normaloff>:/help-circle</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="doneButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="type" stdset="0">
         <string notr="true">primary</string>
        </property>
        <property name="dimension" stdset="0">
         <string notr="true">medium</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StalledIssuesView</class>
   <extends>QTreeView</extends>
   <header>StalledIssuesView.h</header>
  </customwidget>
  <customwidget>
   <class>StalledIssueTab</class>
   <extends>QFrame</extends>
   <header>StalledIssueTab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../gui/Resources_common.qrc"/>
  <include location="../../../gui/Resources_light.qrc"/>
 </resources>
 <connections/>
</ui>
